app/components/start_page_banner/style.scss

```diff
@@ -1,7 +1,41 @@
 @import "../base.scss";
 @import "../../webpacker/styles/inverted";
 
+.app-phase-banner-env--qa .govuk-tag {
+  background-color: govuk-colour("orange");
+}
+
+.app-phase-banner-env--staging .govuk-tag {
+  background-color: govuk-colour("red");
+}
+
+.app-phase-banner-env--sandbox .govuk-tag,
+.app-phase-banner-env--review .govuk-tag {
+  background-color: govuk-colour("purple");
+}
+
+.app-header-env--qa,
+.app-header-env--qa .govuk-header__container {
+  border-color: govuk-colour("orange");
+}
+
+.app-header-env--staging,
+.app-header-env--staging .govuk-header__container {
+  border-color: govuk-colour("red");
+}
+
+.app-header-env--sandbox,
+.app-header-env--review,
+.app-header-env--sandbox .govuk-header__container,
+.app-header-env--review .govuk-header__container {
+  border-color: govuk-colour("purple");
+}
+
```

Comment:
I wonder if the rules about each environment could be kept together - or even combined so there's multiple selectors, but one rule.